share/nmt_adder.go

```diff
@@ -7,26 +7,26 @@ import (
 	"github.com/ipfs/go-merkledag"
 
 	"github.com/ipfs/go-cid"
-	ipld "github.com/ipfs/go-ipld-format"
+	format "github.com/ipfs/go-ipld-format"
```

Comment:
If we move this file to `ipld`(main review message) then we can keep this mport as `ipld`